.playbook.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Ansibox will prepare you
+  hosts: localhost
+  connection: local
+
+  roles:
+    - { role: base_osx, when: "ansible_os_family == 'Apple'"  }
+    - { role: base_fedora, when: "ansible_distribution == 'Fedora'"  }
+    - dotfiles
```

Comment:
I'd really like to be able to dynamically fetch and use roles. We could provide a couple, but instead of this file being static, I think we should template it out from a first stub playbook. That stub playbook would perhaps clone a few things gather info, template out another playbook full of roles, and then a second execution of ansible-playbook would run on the templated file. Since there is a shell script to run here we can do two playbook calls.